src/lib/utils/thorchain/memo/addFinalAssetLimitToMemo.ts

```diff
@@ -0,0 +1,70 @@
+import { assertAndProcessMemo } from './assertAndProcessMemo'
+import { MEMO_PART_DELIMITER } from './constants'
+
+type AddLimitToMemoArgs = {
+  /**
+   * A THORChain memo, with or without a limit/MinOut.
+   */
+  memo: string
+  /**
+   * The final asset amount out to be added to the memo. THOR Aggregator will use the last two numbers as exponents.
+   */
+  finalAssetAmountOut: string
+}
+
+/**
+ * A simple util that adds a final asset limit to a THORChain memo, if the memo doesn't already have one.
+ */
+export const addFinalAssetLimitToMemo = ({ memo, finalAssetAmountOut }: AddLimitToMemoArgs) => {
```

Comment:
nitpick: we could standardize the limit/MinOut terminology across this domain (i.e discrepancy between `finalAssetLimit` and `finalAssetAmountOut` terminologies in this line)

Reply:
Changed to finalAssetLimit everywhere in [2a6fc6a](https://github.com/shapeshift/web/pull/7146/commits/2a6fc6a57521f17fb2fa9ee67d3d46f222c6e53c)